### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by EzequielBurgosL**

### CODE REVIEW

File: staff/ezequielburgos/cv-with-styles/index.html [hunk 1]
```diff
@@ -0,0 +1,145 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="stylesheet" href="styles/main.css">
+    <title>Document</title>
```
Comment (agandia9) [inline staff/ezequielburgos/cv-with-styles/index.html:9]: default title? change it :)

File: staff/ezequielburgos/cv-with-styles/styles/main.css [hunk 2]
```diff
@@ -0,0 +1,43 @@
+@import "desktop.css" screen and (min-width: 700px);
+@import "mobile.css" screen and (max-width: 700px);
+
+
+html{
+    margin: auto;
+    /* background-image: url("https://1.bp.blogspot.com/_nbADpD65WD4/TCT-o6l3aTI/AAAAAAAAgXw/r7rEArUSfQE/w1200-h630-p-k-no-nu/aurora-ultimate-1600x1200.jpg"); */
+    /* background-repeat: no-repeat; */
+    /* background-size: cover; */
```
Comment (agandia9) [inline staff/ezequielburgos/cv-with-styles/styles/main.css:9]: delete comments here!

File: staff/ezequielburgos/cv-with-styles/styles/desktop.css [hunk 3]
```diff
@@ -0,0 +1,93 @@
+*{
```
Comment (manuelbarzi) [inline staff/ezequielburgos/cv-with-styles/styles/desktop.css:1]: TIP move common styles that repeat in mobile and desktop to main.css

File: staff/ezequielburgos/portfolio/index.html [hunk 4]
```diff
@@ -0,0 +1,57 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="stylesheet" href="styles/main.css">
+    <title>Document</title>
```
Comment (MuyZan) [inline staff/ezequielburgos/portfolio/index.html:9]: Put a nice title ;)

**Review by MuyZan — APPROVED** [review]
NICE!!! Very nice!! Just put a title mate! :D

File: staff/ezequielburgos/js/challenges/count-chars/scripts/countChars.js [hunk 5]
```diff
@@ -0,0 +1,28 @@
+// function countChars(str){
```
Comment (Asix94) [inline staff/ezequielburgos/js/challenges/count-chars/scripts/countChars.js]: PLEASE !! remove comments

File: staff/ezequielburgos/js/challenges/count-chars/scripts/main.js [hunk 6]
```diff
@@ -0,0 +1,43 @@
...
+//     console.log('countChars([]) should throw an error', err !== undefined, err);
+// }
+
+
+// count = countChars('abracadabra', function(c) { return c === 'c'; });
+
+// console.log('countChars("abracadabra") should return 5', count === 5, count);
+
+
+// try {
```
Comment (Asix94) [inline staff/ezequielburgos/js/challenges/count-chars/scripts/main.js]: CAUTION with comments !!

File: staff/ezequielburgos/app-template/styles/landing.css [hunk 7]
```diff
@@ -0,0 +1,152 @@
...
+a:hover {
+    color: orange;
+}
+
+main {
+    display: flex;
+    flex-direction: column;
+    width: 100%;
+    height: 75%;
+    /* content: ""; */
```
Comment (agandia9) [inline staff/ezequielburgos/app-template/styles/landing.css:60]: mmmm?

File: staff/ezequielburgos/js/challenges/scripts/to-random-case.js [hunk 8]
```diff
@@ -0,0 +1,20 @@
+function toRandomCase(str) {
```
Comment (agandia9) [inline staff/ezequielburgos/js/challenges/scripts/to-random-case.js:3]: use strict!

File: staff/ezequielburgos/js/challenges/scripts/to-camel-case.js [hunk 9]
```diff
@@ -0,0 +1,37 @@
+function toCamelCase(str) {
```
Comment (agandia9) [inline staff/ezequielburgos/js/challenges/scripts/to-camel-case.js:17]: use strict!

File: staff/ezequielburgos/js/challenges/scripts/reduce-recursive.js [hunk 10]
```diff
@@ -0,0 +1,22 @@
+// REDUCE RECURSIVE:
```
Comment (agandia9) [inline staff/ezequielburgos/js/challenges/scripts/reduce-recursive.js:3]: use strict!

File: staff/ezequielburgos/js/challenges/scripts/reduce-recursive-test.js [hunk 11]
```diff
@@ -0,0 +1,6 @@
+a = [1, 2, 5, 6, 7]
```
Comment (agandia9) [inline staff/ezequielburgos/js/challenges/scripts/reduce-recursive-test.js]: use strict!

File: staff/ezequielburgos/js/challenges/scripts/for-each-recursive.test.js [hunk 12]
```diff
@@ -0,0 +1,38 @@
+// forEach(a, console.log)
```
Comment (agandia9) [inline staff/ezequielburgos/js/challenges/scripts/for-each-recursive.test.js:3]: use strict!

File: staff/ezequielburgos/js/challenges/scripts/for-each-recursive.js [hunk 13]
```diff
@@ -0,0 +1,23 @@
+// RECURSIVIDAD
```
Comment (agandia9) [inline staff/ezequielburgos/js/challenges/scripts/for-each-recursive.js:3]: use strict!

File: staff/ezequielburgos/js/challenges/scripts/cube.test.js [hunk 14]
```diff
@@ -0,0 +1,2 @@
+var numberToCube = cube([12,3,4,5,6,7]);
```
Comment (agandia9) [inline staff/ezequielburgos/js/challenges/scripts/cube.test.js]: use strict!

File: staff/ezequielburgos/js/challenges/scripts/cube.js [hunk 15]
```diff
@@ -0,0 +1,15 @@
+function cube(num) {
```
Comment (agandia9) [inline staff/ezequielburgos/js/challenges/scripts/cube.js]: use strict!

File: staff/ezequielburgos/js/challenges/scripts/countWords.test.js [hunk 16]
```diff
@@ -0,0 +1,3 @@
+var count = countWords('Hello World');
```
Comment (agandia9) [inline staff/ezequielburgos/js/challenges/scripts/countWords.test.js:3]: use strict!

File: staff/ezequielburgos/js/challenges/scripts/countWords.js [hunk 17]
```diff
@@ -0,0 +1,5 @@
+function countWords(str){
```
Comment (agandia9) [inline staff/ezequielburgos/js/challenges/scripts/countWords.js]: use strict!

File: staff/ezequielburgos/js/challenges/scripts/to-random-case.test.js [hunk 18]
```diff
@@ -0,0 +1,47 @@
+
```
Comment (agandia9) [inline staff/ezequielburgos/js/challenges/scripts/to-random-case.test.js:3]: use strict!

File: staff/ezequielburgos/js/challenges/scripts/to-roman-numeral.js [hunk 19]
```diff
@@ -0,0 +1,46 @@
+function toRomanNumeral(num) {
```
Comment (agandia9) [inline staff/ezequielburgos/js/challenges/scripts/to-roman-numeral.js:16]: use strict!

File: staff/ezequielburgos/js/challenges/scripts/to-random-case.test.js [hunk 20]
```diff
@@ -0,0 +1,49 @@
...
+var input = 'Hello World';
+
+var output = toRandomCase(input);
+
+console.log('toRandomCase(input)')
+
+
+// testing our code:
+'use strict';
```
Comment (Asix94) [inline staff/ezequielburgos/js/challenges/scripts/to-random-case.test.js:12]: Why 2 use strict in the same archive

File: staff/ezequielburgos/js/challenges/scripts/unit-testing-tool.js [hunk 21]
```diff
@@ -33,4 +33,73 @@ function runWithErrorHandling(func) {
...
+
+//         try {
+//             testCase();
+//         } catch (err) {
+//             error = err;
+//         }
+
+//         return error;
+//     };
+// }
```
Comment (agandia9) [inline staff/ezequielburgos/js/challenges/scripts/unit-testing-tool.js]: clean OR review comments...!

File: staff/ezequielburgos/cv-with-styles/styles/desktop.css (hunk 3 above)
Comment (manuelbarzi) [reply staff/ezequielburgos/cv-with-styles/styles/desktop.css:1]: APPLY changes, please

**Comment by manuelbarzi** [comment]
ATTEND your pr and APPLY changes, please

**Comment by manuelbarzi** [comment]
merging, but please, apply the pending changes anyway on next pr.

**Pull request merged**